components/history/Filter.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/styles';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+export default function Filter(props) {
+  const classes = useStyles();
+  const { filterOptions, onChange } = props;
+
+  return (
+    <div>
+      <FormControl>
```

Comment:
I don't believe the `<div>` is necessary

Reply:
removed